test/metabase/models/secret_test.clj

```diff
@@ -1,20 +1,30 @@
 (ns metabase.models.secret-test
-  (:require [clojure.test :refer :all]
+  (:require [clojure.java.io :as io]
+            [clojure.test :refer :all]
             [metabase.models.secret :as secret :refer [Secret]]
             [metabase.test :as mt]
-            [metabase.util.encryption-test :as encryption-test]))
+            [metabase.util.encryption-test :as encryption-test])
+  (:import [java.io DataInputStream File]
+           java.nio.charset.StandardCharsets))
+
+(defn- value-matches?
+  "Returns true iff `expected` value matches the `actual` (bytes) value. If `expected` is a String, then `actual` is
```

Comment:
s/iff/if

Reply:
"iff" = "if and only if", which I think is correct here.